core/app/modules/kit-library/assets/js/components/error-screen.scss

```diff
@@ -8,5 +8,6 @@
 	&-description {
 		color: tints( 500 );
 		text-align: center;
+		max-width: 520px;
```

Comment:
Why are u using an explicit `px` value? Is that from the design?
Maybe you could use `vw` or something?

Reply:
I don't want to use `vw` because on mobile I want it to be full width.